docs/learning-powershell/powershell-beginners-guide.md

```diff
@@ -96,12 +96,12 @@ Alias           type -> Get-Content
 …
 ```
 
-As you can see `cls` is an alias of `Clear-Host`.
+As you can see `cls` or `clear` is an alias of `Clear-Host`.
 
 Now try it:
 ```powershell
 PS /> Get-Process
-PS /> cls
+PS /> cls or clear
```

Comment:
It is command line example, we can not execute `cls or clear`.

Reply:
Fixed. Comment above and `Get-Alias` output send the message across, so removed ' or clear' from this example.